test/functional/test_runner.py

```diff
@@ -703,7 +704,7 @@ def was_successful(self):
 def check_script_prefixes():
     """Check that test scripts start with one of the allowed name prefixes."""
 
-    good_prefixes_re = re.compile("^(example|feature|interface|mempool|mining|p2p|rpc|wallet|tool)_")
+    good_prefixes_re = re.compile("^(example|feature|fee|interface|mempool|mining|p2p|rpc|wallet|tool)_")
```

Comment:
I'm not sure this is a good prefix. Maybe `feature_feeestimates_persist`?

Reply:
Yep, sounds reasonable, done